sources/sections-a1-en/13-13th-cgpm.adoc

```diff
@@ -30,7 +30,7 @@ ____
 [[cgpm13th1967r3]]
 === SI unit of thermodynamic temperature (kelvin) (CR, 104 and _Metrologia_, 1968, 4, 43)(((International Temperature Scale of 1990 (ITS-90))))(((kelvin (stem:["unitsml(K)"]))))(((thermodynamic temperature)))
 
-NOTE: At its 1980 meeting, the CIPM approved the report of the 7th meeting of the CCU, which requested that the use of the symbols "stem:["unitsml(degK)"]" and "stem:["unitsml(deg)"]" no longer be permitted.
+NOTE: At its 1980 meeting, the CIPM approved the report of the 7th meeting of the CCU, which requested that the use of the symbols "stem:["unitsml(degK)"]" and "stem:["deg"]" no longer be permitted.
```

Comment:
If `degK` is encoded as a unit, `deg` is certainly a unit too. This change seems wrong.

Reply:
@ronaldtse I disagree, "deg" is shown as literal text in original document:

![issue1](https://user-images.githubusercontent.com/33627611/122430634-c5aa9400-cf61-11eb-88e8-7eaf2a695d0e.PNG)

So, I encoded it as literal text.

Reply:
OK